puppet/zulip/manifests/nginx.pp

```diff
@@ -95,6 +95,13 @@
     mode   => '0650'
   }
 
```

Comment:
@timabbott Any reason why this won't work?

Reply:
This is good, though I think the variable should be called `cert_auto_renew`.  I'll just change that before merging.